docs/docs/03-interactivity-and-dynamic-uis.md

```diff
@@ -11,22 +11,26 @@ You've already [learned how to display data](/react/docs/displaying-data.html) w
 ## A Simple Example
 
 ```javascript
-var LikeButton = React.createClass({
-  getInitialState: function() {
-    return {liked: false};
-  },
-  handleClick: function(event) {
+class LikeButton extends React.Component {
+  constructor() {
+    super();
```

Comment:
Yeah, I thought about that.  But if we're always going to pass props, why not context?

Also, what's the benefit of context, when 95% of the time, you don't need to access props.

I convinced myself that it was better to only pass them/use when you need them.